core/src/components/datetime/datetime.scss

```diff
@@ -357,6 +357,11 @@ ion-picker-column-internal {
 }
 
 :host .calendar-day:after {
+  content: " ";
+}
+
+:host .calendar-day:after,
+:host .calendar-day-event-highlight {
```

Comment:
Do you mean change the color of the focus style to match the highlight's BG color? I'm hesitant to do that because then the focus highlight would no longer always match the color of a selected date. (And the design doc says that the highlight shouldn't override the selected date styles.) Having the focus highlight change color as you navigate around with the keyboard would also feel strange IMO.

Reply:
I'm referring to dates that have the highlight styles applied: 
![image](https://user-images.githubusercontent.com/2721089/218556385-4754ebe4-5217-41f8-b057-355778fa2dcd.png)

In MD, you can focus a non-selected button. However, it is using the default color palette so the blue focus state ends up mixing with the red background that was specified.

Reply:
As opposed to something like this: 
![image](https://user-images.githubusercontent.com/2721089/218556602-6558e317-5b38-417c-a149-51e2e8140085.png)

Reply:
(Notice how the red color in the first image is slightly mixed with the blue)

Reply:
I've updated it so that the focus state matches the background color of the date (if one is specified) using an opacity of 0.2, on both MD and iOS. I don't love it, though, for a couple reasons:
1. If you use a lighter background color for the highlight, the focus state is hard to see.
2. The color of your "cursor" changes as you navigate around using the keyboard, which feels strange IMO.

We could solve item 1 by letting the developer set the opacity on a per-highlight basis, but that feels overly complicated. I would rather either keep the old behavior (where the focus state is always that light blue, or whatever the base color is set to, and overlays the date), or maybe have the focus color sit behind the highlight BG instead so you just get a light blue ring around it, rather than the colors mixing together. Have a look and let me know what you think.

Reply:
I like having the focus ring sit behind the background. Honestly, I don't know why it did not already do that. Maybe an old bug that we're noticing?

Reply:
The (default) focus ring and selected date circle are the same color, so before, there wouldn't be any visual difference if it were in front of or behind the background, since the selected date style is 100% opacity anyway.

Either way, I've updated it again; all styles are now contained within the `calendar-day-highlight` element, and the focus style always uses the default light blue, showing as a ring around the highlight in MD mode instead of overlaying it.

Note that in iOS, I added an `!important` to the focus/active background color to overwrite the highlight background (which is inline). The only way I can think of to get around this (while avoiding the focus and highlight colors blending together) would be to use an :after with 100% opacity that covers up the highlight, but then we'd need to plug in a separate color that looks like the default blue at 20% opacity, which means you wouldn't be able to customize it easily. 🤔

Reply:
I like having the focus ring behind the element so we can avoid the blending. Thanks for fixing that.

I'm fine having the focus state default to the component's color palette. This wasn't a behavior that was scoped, so I think it would be fine to keep it as you have it and see what the developer feedback is.